mainwindow.cc

```diff
@@ -157,7 +157,8 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
   }
 
   wuri = new WebUrlRequestInterceptor();
-  QWebEngineProfile::defaultProfile()->setUrlRequestInterceptor(wuri);
+  QWebEngineProfile::defaultProfile()->setUrlRequestInterceptor( wuri );
```

Comment:
I am not sure if using a different profile is not a good idea. But perhaps you could use [QWebEnginePage::acceptNavigationRequest](https://doc.qt.io/qt-5/qwebenginepage.html#acceptNavigationRequest) instead of the interceptor here and get rid of the issue?